source/iface/host-task-add-to-handler.rst

```diff
@@ -5,3 +5,31 @@
 **********************************
 Additions to the ``sycl::handler``
 **********************************
+
+This section describes member functions in the command group handler
+class that are used with host tasks.
+
+::
+
+  class handler {
+    ...
+
+        public
+        : template <typename T>
+          void
+          host_task(T&& hostTaskCallable); // (1)
+
+    ...
+  };
+
+Effects: Enqueues an implementation-defined command to the SYCL runtime
+to invoke ``hostTaskCallable`` exactly once. The scheduling of the
+invocation of ``hostTaskCallable`` in relation to other commands
+enqueued to the SYCL runtime must be in accordance with the dependency
+model described in |SYCL_SPEC_MEM_MODEL|. Initializes an
+``interop_handle`` object and passes it to ``hostTaskCallable``
+when it is invoked if ``std::is_invocable_v<T, interop_handle>``
+evaluates to true, otherwise invokes ``hostTaskCallable``
+as a nullary function.
+
+.. seealso:: :ref:`handler`
```

Comment:
Should this link to the SYCL spec?  Currently links within the SYCL reference.

Reply:
Added only link to SYCL spec with https://github.com/KhronosGroup/SYCL_Reference/pull/153/commits/8322c7638e0e30a13a089dd315d66ee179e33738